lib/ansible/modules/storage/netapp/na_ontap_nfs.py

```diff
@@ -419,15 +480,27 @@ def state_changed(expected, current):
         def is_modify_needed():
             if (((self.nfsv3 is not None) and state_changed(self.nfsv3, nfs_service_details['is_nfsv3_enabled'])) or
                 ((self.nfsv3_fsid_change is not None) and state_changed(self.nfsv3_fsid_change, nfs_service_details['is_nfsv3_fsid_change_enabled'])) or
+                ((self.nfsv4_fsid_change is not None) and state_changed(self.nfsv4_fsid_change, nfs_service_details['is_nfsv4_fsid_change_enabled'])) or
                 ((self.nfsv4 is not None) and state_changed(self.nfsv4, nfs_service_details['is_nfsv40_enabled'])) or
                 ((self.nfsv41 is not None) and state_changed(self.nfsv41, nfs_service_details['is_nfsv41_enabled'])) or
+                ((self.nfsv41_pnfs is not None) and state_changed(self.nfsv41_pnfs, nfs_service_details['is_nfsv41_pnfs_enabled'])) or
+                ((self.nfsv4_numeric_ids is not None) and state_changed(self.nfsv4_numeric_ids, nfs_service_details['is_nfsv4_numeric_ids_enabled'])) or
                 ((self.tcp is not None) and state_changed(self.tcp, nfs_service_details['is_tcp_enabled'])) or
                 ((self.udp is not None) and state_changed(self.udp, nfs_service_details['is_udp_enabled'])) or
                 ((self.nfsv40_acl is not None) and state_changed(self.nfsv40_acl, nfs_service_details['is_nfsv40_acl_enabled'])) or
+                ((self.nfsv40_read_delegation is not None) and state_changed(self.nfsv40_read_delegation,
+                                                                             nfs_service_details['is_nfsv40_read_delegation_enabled'])) or
                 ((self.nfsv40_write_delegation is not None) and state_changed(self.nfsv40_write_delegation,
                                                                               nfs_service_details['is_nfsv40_write_delegation_enabled'])) or
-                ((self.nfsv40_write_delegation is not None) and state_changed(self.nfsv40_write_delegation,
```

Comment:
Not sure why this is showing as changed.

Reply:
Hi, I've added version_added: ‘2.9’ to new module options. 

The code piece around ```self.nfsv40_write_delegation is not None``` is just a code alignment. 

There's a bunch of [PyLint](https://www.pylint.org/) problems. I'll try to re-factor the code later. ''' def is_modify_needed(): ''' is the first candidate.

 So far, I've just added new options to module.

Reply:
BTW: The shippable error was cause by repo clone problem - ```fatal: Couldn't find remote ref pull/59505/head```